### DISCUSSION TITLE
Can't get a led blink

### DISCUSSION THREAD

**Discussion by matou78** [SpenceKonde/DxCore#262] [Q&A]
Hello all, 
Sorry if it seems like a shit question, but can't find a solution alone.
I'm trying to play with some AVR64DA32 chips but can't do anything with them .

I'm using platformIO ( sometimes Arduino IDE if not sure about settings), 
I made a little quick program to test my chips but I'm not able to get anything from it .

![image](https://user-images.githubusercontent.com/20770243/157882591-2595c7de-f994-472d-afee-f4f0ffeaa901.png)

I don't get anything on Serial monitor or any blinking LED ,
It compiles and write to the chip, but nothing happens .

Thanks for your time

**Comment by mikrocoder** [comment]
Formally first nothing wrong, but do not know platformIO. Try once without Serial.

**Comment by matou78** [comment]
The first thing i did, was just the led blink, then i added the serial to "debug" and see if at least this was working and not something i did wrong .
I probed the pads of the chip and some are at 0V and some at 0.5V ...
Even the one that i want to use for the LED is at 0.5V not at 0 or 5/3.3V .
Need to try with analogWrite too but did not have time yesterday :(

I will try to change the chip and try again . Maybe something wrong with this chip ?

**Comment by mikrocoder** [comment]
Try direct register programming first. Without Arduino functions. Not that an assignment in the lib is wrong.

Example
```cpp
#include <util/delay.h>

void setup (void)
{
   PORTF_DIR = PIN5_bm;
}

void loop (void)
{
  PORTF_OUTTGL = PIN5_bm;
  _delay_ms(500);
}
```

**Comment by matou78** [comment]
Well , looks like by miracle, today everything is working lol
Tried this morning by programming again through PlatformIO and ArduinoIDE but same thing, nothing working

I then changed the chip, programmed again through ArduinoIDE and it's working again.

Even the chip 74HC4052D to change between UPDI and Serial is working *_*

The only difference is in versions of DXcore, ArduinoIDE i'm still on 1.3.10 and platformIO has the 1.4.7 if not wrong
No idea why i can't update it in ArduinoIDE but well, for now i can play some with it .

Need to find what is wrong in PlatformIO now,

Thanks for your help mikrocoder

EDIT 1:
Updated the Dxcore on ArduinoIDE and it's still working

Something is bad in the firmware for the AVRxxDA in platformIO , even uploading through uart it's not working.
Well , going to cry about this to folks in PlatformIO, maybe they will have some ideas :)

**Reply by SpenceKonde** [reply]
I am very conufsed as to the nature of the issue at this point in your investigation, I think I need a summary to hope to make sense of what you've observed
But first i needd t o get 1.4.10 out, because what I was going to release fails to build on the majority of parts in any way.

**Reply by matou78** [reply]
thanks for your response :)

Well , i tried a lot of things this last 2 weeks, i tracked the issue to the platformIO integration of DxCore ...
Things are missing and few things needs to be fixed .

i link you the issues i ended opening in platformIO
https://github.com/platformio/platform-atmelmegaavr/issues/41
https://github.com/platformio/platform-atmelmegaavr/issues/42

Using DxCore in arduinoIDE works as it should, it's just that since i started using platformIO , hard to get back into arduinoIDE :p
